### ISSUE TITLE
Separate clients for each facet

### ISSUE THREAD

**Issue by IgnisDa** [IgnisDa/ryot#184]
Going by the poll i conducted on [Reddit](https://www.reddit.com/r/selfhosted/comments/14rz77l/what_is_more_preferable_for_a_selfhosted_tracker/?utm_source=share&utm_medium=mweb), I will be creating separate apps for media and fitness. Keeping this issue here so that others can also chime in.

**Comment by duplaja** [comment]
One of the things that originally drew me to ryot was the ability to have everything in one place. I feel like this is getting away from that, unfortunately. I really liked the idea of not having 3-5+ subdomains to keep track of things I want to track.

**Comment by IgnisDa** [comment]
Makes sense, but won't cramming everything together on one website make things too complex. What can be done to fix that?

**Comment by duplaja** [comment]
I like the idea of (what there seems to be currently) of being able to toggle categories on and off. and maybe a home page / panel with just those categories being shown. This is especially true with the "Allow Ryot to track anything" idea that is also pinned, so we don't have to spin up a separate instance for each.

That's from a user standpoint, of course. From a code-base standpoint, I'm not sure which would be easier for you.

**Comment by IgnisDa** [comment]
How about a sidebar with "Media", "Fitness" (one for each facet)? When you visit fitness, your location is saved and the next time you open the client, it automatically navigates there.

**Comment by IgnisDa** [comment]
And maybe a corresponding setting which can disable this behaviour.

**Comment by duplaja** [comment]
I'd be open to something like that. And yeah, maybe a toggle / setting to choose which facets to show or hide. Would allow for the introduction of either a "Custom" facet, or the addition of new facets.

**Comment by IgnisDa** [comment]
Makes sense. Thanks for feedback!

**Comment by DieselTech** [comment]
I was also drawn in by the '1 tracker for everything' aspect. I don't want to keep spinning up multiple instances of what is essentially the same software just to track all the different aspects of things. Especially when it would be the same code base but just listening for different "style" events.

**Comment by IgnisDa** [comment]
Thanks for the feedback!
